src/analyzer.ts

```diff
@@ -117,16 +117,29 @@ export class Analyzer {
   }
 
   /**
+   * Returns `true` if the provided resolved URL can be loaded.
```

Comment:
Thanks for documenting these.

Please also mention on each of them that the semantics are determined by the configured url loader/resolver, and that only resolved urls should ever be loading.